components/o-toggle/main.js

```diff
@@ -5,6 +5,6 @@ const constructAll = () => {
 	document.removeEventListener('o.DOMContentLoaded', constructAll);
 };
 
-document.addEventListener('o.DOMContentLoaded', constructAll);
+if (document) document.addEventListener('o.DOMContentLoaded', constructAll);
```

Comment:
```suggestion
if (typeof document !== 'undefined') {
	document.addEventListener('o.DOMContentLoaded', constructAll);
}
```

maybe something like this

Reply:
Updated as per this suggestion.